system/web/context/RequestContext.cfc

```diff
@@ -968,6 +968,7 @@ component serializable=false accessors="true"{
         return arrayToList( [
             isSSL() ? "https://" : "http://",
             CGI.SERVER_NAME,
+			listLen( variables.SESBaseURL, ':' ) == 3 ? ":" & CGI.SERVER_PORT : "",
```

Comment:
Why 3?  Shouldn't it be if it finds two `:` in the URL?

Reply:
Also, modules could be leverages that use the `:` notation. SO it could basically give you a false positive.

I think this should determine the port qualifier to be in certain numbers just to be safe.

```
listFind( "80,443", cgi.server_port ) ? "" : cgi.server_port
```

Reply:
Good point about modules using the : notation.

The solution of looking for ports 80 or 443 could work as well since those are typically omitted.  Another idea would be to use a regex to check for the existence of a port number in the url: `(https?://.*):(\d*)\/?(.*)` and if true, show the port number.
https://www.regextester.com/99331

Reply:
Darn, that regex will still false-positive on the following url: http://www.domain.com/test/:module/.  I'll see if I can find one that will match just the port if present.

Reply:
I think it would be best to jsut see if the server port is within the list

Reply:
Using a predefined list sounds good to me.  While technically http://127.0.0.1:80/ and https://127.0.0.1:443/ are perfectly valid, there's no real reason to include those ports in the URL.

This fix should address the issues impacting people using non-standard ports (like during development)  http://127.0.0.1:52606/